guides/common/modules/ref_host-specific-variables.adoc

```diff
@@ -8,72 +8,72 @@ Note that job templates accept only `@host` variables.
 Returns an interface object.
-|@host.ptable |The partition table name.
-|@host.puppet_ca_server *Deprecated* Use the `host_puppet_ca_server` variable instead. |The Puppet CA server the host must use.
-|@host.puppetmaster *Deprecated* Use the `host_puppet_server` variable instead. |The Puppet server the host must use.
-|@host.pxe_build? |Returns `true` if the host is provisioned using the network or PXE.
-|@host.shortname |The short name of the host.
-|@host.sp_ip |The IP address of the BMC interface.
-|@host.sp_mac |The MAC address of the BMC interface.
-|@host.sp_name |The name of the BMC interface.
-|@host.sp_subnet |The subnet of the BMC network.
-|@host.subnet.dhcp |Returns `true` if a DHCP proxy is configured for this host.
-|@host.subnet.dns_primary |The primary DNS server of the host.
-|@host.subnet.dns_secondary |The secondary DNS server of the host.
-|@host.subnet.gateway |The gateway of the host.
-|@host.subnet.mask |The subnet mask of the host.
-|@host.url_for_boot(:initrd) |Full path to the initrd image associated with this host.
+|`@host.ptable` |The partition table name.
+|`@host.puppet_ca_server` *Deprecated* Use the `host_puppet_ca_server` variable instead. |The Puppet CA server the host must use.
+|`@host.puppetmaster` *Deprecated* Use the `host_puppet_server` variable instead. |The Puppet server the host must use.
+|`@host.pxe_build?` |Returns `true` if the host is provisioned using the network or PXE.
+|`@host.shortname` |The short name of the host.
+|`@host.sp_ip` |The IP address of the BMC interface.
+|`@host.sp_mac` |The MAC address of the BMC interface.
+|`@host.sp_name` |The name of the BMC interface.
+|`@host.sp_subnet` |The subnet of the BMC network.
+|`@host.subnet.dhcp` |Returns `true` if a DHCP proxy is configured for this host.
+|`@host.subnet.dns_primary` |The primary DNS server of the host.
+|`@host.subnet.dns_secondary` |The secondary DNS server of the host.
+|`@host.subnet.gateway` |The gateway of the host.
+|`@host.subnet.mask` |The subnet mask of the host.
+|`@host.url_for_boot(:initrd)` |Full path to the initrd image associated with this host.
 Not recommended, as it does not interpolate variables.
-|@host.url_for_boot(:kernel) |Full path to the kernel associated with this host.
+|`@host.url_for_boot(:kernel)` |Full path to the kernel associated with this host.
 Not recommended, as it does not interpolate variables, prefer boot_files_uri.
-|@provisioning_type |Equals to 'host' or 'hostgroup' depending on type of provisioning.
-|@static |Returns `true` if the network configuration is static.
-|@template_name |Name of the template being rendered.
-|grub_pass |Returns a bootloader argument to set the encrypted bootloader password, such as *--md5pass=#{@host.grub_pass}*.
-|ks_console |Returns a string assembled using the port and the baud rate of the host which can be added to a kernel line.
+|`@provisioning_type` |Equals to 'host' or 'hostgroup' depending on type of provisioning.
```

Comment:
I have not found a way to exclude `'hostgroup'` in Vale without sacrificing all "hostgroup" instances.

Reply:
Would it work to make it code blocks? Since those are really the verbatim values you can expect
```suggestion
|`@provisioning_type` |Equals to `host` or `hostgroup` depending on type of provisioning.
```

Reply:
Yes, this mitigates Vale warnings.